storage/src/validator.rs

```diff
@@ -427,12 +436,12 @@ impl<T: TransactionScheme + Send + Sync, P: LoadableMerkleParameters, S: Storage
             }
             component_sender.send(ValidatorAction::RegisterTxComponent(COL_DIGEST, tx_digest)).unwrap();
 
-            let tx_memo = tx.memorandum();
-            if !self.contains_memo(tx_memo) {
+            let tx_memo = to_bytes![tx.memorandum()].unwrap();
```

Comment:
I'm assuming this can't fail?

Reply:
correct: https://github.com/AleoHQ/snarkVM/blob/v0.5.4/utilities/src/bytes.rs#L129-L148